src/components/organisms/AssetActions/Compute/index.tsx

```diff
@@ -288,6 +288,12 @@ export default function Compute({
         algorithmAssetOrderId,
         selectedAlgorithmAsset.dataToken
       )
+
+      if (!respone) {
+        setError('Error starting compute job.')
```

Comment:
would be great to get after this line also this pattern in here:

```js
Logger.error('[compute] Failed to start job!', error.message)
```

In my error case the body of the response has helpful message under `body.error` so would be great to output this somehow

Reply:
yeah exactly, I was testing now and didn't find a way to get to the error 😃